roles/rook-recovery/README.rst

```diff
@@ -0,0 +1,8 @@
+rook-recovery
+==========
+
+This role is used to deploy the rook jenkins instance in the rook CI aws vpc, The deployied instance will have two ebs volumes were the jobs and the jobs configuration data is located 
```

Comment:
Typo

Reply:
fixed

Reply:
You need to `git add` all the files you fixed then `git commit --amend --no-edit` then `git push -f origin Rook-Recovery-Role` to push the fixes.

Reply:
I did it yesterday but i forgot doing git add, now it should be fine